### PR TITLE
Changed certs-tool documentation

### PR DESCRIPTION
**Pull request by c-bordon**
## Description
Changes are made in the documentation taking into account the latest changes made in the certs-tools tool

Screenshot_20220323_112430

Screenshot_20220323_112502

## Checks
- [x] It compiles without warnings.
- [ ] Spelling and grammar. 
- [ ] Used impersonal speech. 
- [ ] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

## Note to the reviewer

This PR includes changes to the `redirect.js` script that need to be included in all production branches.

### CODE REVIEW

File: source/user-manual/certificates.rst [hunk 1]
```diff
@@ -67,11 +64,11 @@ After running the script, the directory ~/certs will be created and will have th
         certs/
```
Comment (alberpilot) [inline source/user-manual/certificates.rst:64]: This folder is `wazuh-certificates` now.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: source/user-manual/certificates.rst (hunk 1 above)
Comment (c-bordon) [reply source/user-manual/certificates.rst]: Done: https://github.com/wazuh/wazuh-documentation/commit/3c5a90c58931d83484826b0f85c8e0c657ce2d18

**Review by s-ocando — COMMENTED** [review]
@c-bordon please make the following changes:
 
- Update the tool name from ``wazuh-cert-tool.sh`` to [wazuh-certs-tool.sh](https://packages-dev.wazuh.com/4.3/wazuh-certs-tool.sh). 

- In the `source/user-manual/certificates.rst` document, include the download links for ``wazuh-certs-tool.sh`` and ``config.yml``. 

- Add instructions on how to create certificates with pre-existing ``root-ca.pem`` and ``root-ca.key``.

**Review by s-ocando — APPROVED** [review]

**Pull request merged**